docs/rules/no-restricted-paths.md

```diff
@@ -1,4 +1,6 @@
-# import/no-restricted-paths: Restrict which files can be imported in a given folder
```

Comment:
preserve

Reply:
> In order to prevent such scenarios this rule allows you to define restricted zones where you can forbid files from imported if they match a specific path.

This line from the body might capture this already.

Reply:
Per https://github.com/import-js/eslint-plugin-import/pull/2582#discussion_r1008920390, i think the exact description phrase text needs to appear in the doc somewhere.